pulp_rpm/app/constants.py

```diff
@@ -1,26 +1,17 @@
 from types import SimpleNamespace
 
+from pulpcore.plugin.constants import ALL_KNOWN_CONTENT_CHECKSUMS
+
+# Due to unsorted 'ALL_KNOWN_CONTENT_CHECKSUMS' must be sorted
+# when used or Django would like to create new migration
+
+
 CHECKSUM_TYPES = SimpleNamespace(
-    UNKNOWN="unknown",
```

Comment:
Why did we have `unknown`, does it mean that createrepo_c can return this checksum_type?
FWIW, I think the list of checksums came from createrepo_c.

Reply:
same as comment above

Reply:
same as comment above